uxarray/helpers.py

```diff
@@ -1,91 +1,56 @@
-import os
 import xarray as xr
 from pathlib import PurePath
 
 
-# helper function to find file type
-def determine_file_type(filepath):
-    """Checks file path and contents to determine file type. Supports detection
-    of UGrid, SCRIP, Exodus and shape file.
+def parse_grid_type(filepath, **kw):
```

Comment:
Could you please reflect this function renaming in documentation, docs/user_api/index.rst?

Reply:
Done!